get-started.md

```diff
@@ -1,24 +1,11 @@
 
-The [source code is open](https://github.com/pelias/pelias) to view, modify, and contribute to. It's called Pelias, an experimental, community focused geocoder from Mapzen. Pelias is available for free and licensed for both commercial and non-commercial use.
+Through a process known as [geocoding](/search.md), Mapzen Search allows you to use natural language to find a particular place by entering an address or the name of a landmark or business, and then translates the result in to the geographic coordinates used by computers. Mapzen Search accesses [global databases](/data-sources.md) of place names and locations, but you can receive more locally relevant search results by limiting the search to a particular radius around a location (such as a cell phone), region, or country. With text [autocompletion capabilities](/autocomplete.md), you can search for places and match against Mapzen Search data in real-time. 
 
-With Mapzen Search you can transform the natural language that we use to describe places in to geographic coordinates that computer systems can understand. You can also do the opposite and turn a latitude and longitude values in to a list of nearby places.
+Mapzen Search also enables the opposite workflow, known as [reverse geocoding](/reverse.md), to transform latitude and longitude values in to a list of places. This process attempts to find the name and address of the place nearest a longitude and latitude pair. For example, you can click a postion on the map to learn which business is located there.
 
-Mapzen Search can be used to:
-> - quickly localize your users so they can discover locally relevant content.
-> - provide a super-fast typeahead solution which feels intuitive for non technical users.
-> - improve your database by adding geographic columns, such as `home_city` and `country` to your users table.
-> - store numeric co-ordinate data instead of strings, for later use in reporting, analytics and machine learning.
-> - improve existing geographic data by augmenting it with information about the enveloping `country`, `city` or `state`.
+Behind the scenes of your app, Mapzen Search can improve the quality of your own data by augmenting it with information where your users are located. You can pass the coordinates returned from Mapzen Search to other services, such as [Mapzen Turn-by-Turn](https://mapzen.com/projects/valhalla) that enables navigation and routing directions between points. The coordinate data obtained is numeric, rather than a string type, which facilitates better reporting, analytics, and machine learning. Mapzen Search can also convert a geographic coordinate pair into the administrative boundary hierarchy containing it, such as a from the neighbourhood to the local administrative area, and on up to the country level. Getting this information can be useful when filtering for places in non-geographic databases.
```

Comment:
This is the paragraph you wanted the most feedback on, right?

Everything in it is technically correct, but the sentence about data coming back as numeric rather than strings probably is too much detail this early on in an overview page.